tests/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ else()
     add_custom_target(catch)
 endif()
 
+if (MSVC)
+    if (GSL_CXX_STANDARD EQUAL 17)
```

Comment:
This can be merged with the above to become `MSVC AND GSL_CXX_STANDARD EQUAL 17`.

Reply:
yes thanks